adivinaLaPalabra/src/main/java/com/soprasteria/adivinaLaPalabra/model/RoundEntity.java

```diff
@@ -1,15 +1,21 @@
 package com.soprasteria.adivinaLaPalabra.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import lombok.Data;
 
 import java.util.UUID;
 @Data
-public class Round {
+@Entity
+@Table(name = "round")
+public class RoundEntity {
 
+    @Id
     private UUID id;
```

Comment:
Change this to Long and let de Database automanage the values
```suggestion
    private Long id;
```